tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -271,233 +271,233 @@
     <string name="toastSettings_saved">Settings saved</string>
 
 
-    <string name="pref_dnscrypt_global_settings_categ">GLOBAL SETTINGS</string>
+    <string name="pref_dnscrypt_global_settings_categ">Settings for the whole device</string>
 
-    <string name="pref_advanced"> For advanced user only!</string>
+    <string name="pref_advanced">Only for advanced users!</string>
```

Comment:
This is used for a category title and in summaries, which differ between this and title casing.

Reply:
No difference. Just to warn the user to be careful with these parameters.